lib/postBandName.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+module.exports = function postBandName (word, object) {
+
+ //save band name to bandname object when save button is clicked
```

Comment:
ah! always good to have something fun to work on later. (no impact on the grade, since you submitted your app under the beginner track)